extensions/BMO/template/en/default/bug/create/create-ipc.html.tmpl

```diff
@@ -0,0 +1,267 @@
+  });
+  $('.date_field-img')
+    .click(function(event) {
+      var id = $(event.target).attr('id').replace(/-img$/, '');
+      $('#' + id).datetimepicker('show');
+  });
+  $('#crmForm').submit(function() {
+    $('#short_desc').val($('#short_desc').val() + ': ' + $('#program_name').val());
+  });
+  $('#new_email_request').on('change', function() {
+    if ($('#new_email_request').val() == 'Yes') {
+      $('#email_cadence_row').show();
+      $('input[name="email_cadence"]').each(function() {
+        $(this).attr('required', true);
+      });
+    }
+    else {
+      $('#email_cadence_row').hide();
+      $('input[name="email_cadence"]').each(function() {
+        $(this).removeProp('required');
+      });
+    }
+  });
+});
+[% END %]
+
+[% PROCESS global/header.html.tmpl
+   title              = "IPC Request"
+   generate_api_token = 1
+   style_urls         = [ "skins/standard/attachment.css",
+                          "js/jquery/plugins/datetimepicker/datetimepicker.css" ]
+   style              = inline_style
+   javascript         = inline_javascript
+   javascript_urls    = [ "js/field.js", "js/util.js" ]
+   jquery             = [ "datetimepicker" ]
+%]
+
+<div id="heading">IPC Requests</div>
+<div>
+  <p>
+  You’ve decided to use the snippet channel to drive attention to one of your projects, Awesome! The snippet channel drives hundreds of millions of impressions a month. It’s great tool to support your marketing efforts. Complete this form and we will follow up with you soon (maybe today, but definitely within 3 days.</p>
+  <p>
+    Tell us more (all fields are required):
+  </p>
+
+  <form method="post" action="post_bug.cgi" id="crmForm">
+    <input type="hidden" id="short_desc" name="short_desc" value="New IPC Request">
+    <input type="hidden" name="product" value="Marketing">
+    <input type="hidden" name="component" value="Email">
+    <input type="hidden" name="rep_platform" value="All">
+    <input type="hidden" name="op_sys" value="Other">
+    <input type="hidden" name="version" value="unspecified">
+    <input type="hidden" name="priority" value="--">
+    <input type="hidden" name="bug_severity" value="normal">
+    <input type="hidden" name="format" value="crm">
+    <input type="hidden" name="token" value="[% token FILTER html %]">
+    [% IF user.in_group('canconfirm') %]
+      <input type="hidden" name="bug_status" value="NEW">
+    [% END %]
+
+    <div class="row">
+      <div class="row_desc">
+        <label class="required" for="team_name">
+          <strong>What team are you a part of?</strong>
+        </label>
+      </div>
+      <input required type="text" name="team_name" id="team_name" size="80">
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <label for="user_story">
+          <strong>What is the user story associated with your snippet?</strong>
+        </label>
+      </div>
+      <input type="text" name="user_story" id="user_story" size="80">
```

Comment:
In the google form, this is a long text, so maybe it should be a <textarea> instead of a single-line input? also, if this is named cf_user_story it will go into the user story field on the show bug page.